### ISSUE TITLE
Inviting/Adding user to project fails

### ISSUE THREAD

**Issue by JantsoP** [riotkit-org/docker-taiga#36]
When I'm trying to add new user to my project, I get the following error.
``` taiga_1     | 2020-04-16 08:42:17,616 DEBG 'backend' stderr output:
 taiga_1     | ERROR:2020-04-16 08:42:17,591: Internal Server Error: /api/v1/memberships/bulk_create
 taiga_1     | Traceback (most recent call last):
 taiga_1     |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
 taiga_1     |     response = get_response(request)
 taiga_1     |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
 taiga_1     |     response = self._get_response(request)
 taiga_1     |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
 taiga_1     |     response = self.process_exception_by_middleware(e, request)
 taiga_1     |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
 taiga_1     |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
 taiga_1     |   File "/usr/src/taiga-back/taiga/base/api/viewsets.py", line 106, in view
 taiga_1     |     return self.dispatch(request, *args, **kwargs)
 taiga_1     |   File "/usr/local/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
 taiga_1     |     return view_func(*args, **kwargs)
 taiga_1     |   File "/usr/src/taiga-back/taiga/base/api/views.py", line 460, in dispatch
 taiga_1     |     response = self.handle_exception(exc)
 taiga_1     |   File "/usr/src/taiga-back/taiga/base/api/views.py", line 458, in dispatch
 taiga_1     |     response = handler(request, *args, **kwargs)
 taiga_1     |   File "/usr/src/taiga-back/taiga/projects/api.py", line 1002, in bulk_create
 taiga_1     |     precall=self.pre_save)
 taiga_1     |   File "/usr/src/taiga-back/taiga/projects/services/members.py", line 65, in create_members_in_bulk
 taiga_1     |     db.save_in_bulk(members, callback, precall)
 taiga_1     |   File "/usr/local/lib/python3.6/contextlib.py", line 52, in inner
 taiga_1     |     return func(*args, **kwds)
 taiga_1     |   File "/usr/src/taiga-back/taiga/base/utils/db.py", line 99, in save_in_bulk
 taiga_1     |     callback(instance, created=created)
 taiga_1     |   File "/usr/src/taiga-back/taiga/projects/api.py", line 1078, in post_save
taiga_1     |     services.send_invitation(invitation=object)
taiga_1     |   File "/usr/src/taiga-back/taiga/projects/services/invitations.py", line 34, in send_invitation
taiga_1     |     email.send()
taiga_1     |   File "/usr/local/lib/python3.6/site-packages/django/core/mail/message.py", line 348, in send
taiga_1     |     return self.get_connection(fail_silently).send_messages([self])
taiga_1     |   File "/usr/local/lib/python3.6/site-packages/django/core/mail/backends/console.py", line 36, in send_messages
taiga_1     |     self.write_message(message)
taiga_1     |   File "/usr/local/lib/python3.6/site-packages/django/core/mail/backends/console.py", line 23, in write_message
taiga_1     |     self.stream.write('%s\n' % msg_data)
taiga_1     | UnicodeEncodeError: 'ascii' codec can't encode characters in position 10444-10446: ordinal not in range(128)
```
Tested with already existing user and tried with a dummy user. 
Versions tested:  5.0.10 and 5.0.10-b1.3-RC1

.env file (System is currently a staging environment running on my machine): 
```
# will not allow container to exit even if application exits
VERSION=5.0.10-b1.3-RC1
DEBUG=false

# http settings domain, scheme
TAIGA_HOSTNAME=10.211.55.9
TAIGA_SCHEME=http
TAIGA_PLUGINS=slack

# general
TAIGA_SECRET_KEY=:)
TAIGA_DEFAULT_LOCALE=en

# Database settings
# To use an external database, simply update these and remove the postgres
# service from this docker-compose.yml file
TAIGA_DB_NAME=taigadb
TAIGA_DB_HOST=postgres
TAIGA_DB_USER=postgres
TAIGA_DB_PASSWORD=:)

# To use an external SMTP for emails, fill in these values:
# TAIGA_ENABLE_EMAIL: True
# TAIGA_EMAIL_FROM: no-reply@taiga.mycompany.net
# TAIGA_EMAIL_USE_TLS: True
# TAIGA_EMAIL_HOST: smtp.google.com
# TAIGA_EMAIL_PORT: 587
# TAIGA_EMAIL_USER: me@gmail.com
# TAIGA_EMAIL_PASS: super-secure-pass phrase thing!

# events
TAIGA_ENABLE_EVENTS=false
TAIGA_EVENTS_HOST=events
TAIGA_BROKER_URL=amqp://guest:guest@rabbit:5672
TAIGA_REDIS_URL=redis://redis:6379/0
```

**Comment by JantsoP** [comment]
Solved with going to 5.0.6 but still.... Is .10 just so unstable?

**Comment by blackandred** [comment]
Hi JantsoP, it looks like a bug in the application for me. It has something
to do with the data entered in the application - possibly e-mail
notification content. Possibly some special characters were entered that
cannot be decoded into UTF-8.
I suggest you to look at taiga official repository for already reported
issue, if there is no any existing then open a new one.

**Comment by JantsoP** [comment]
Well I got this issue solved with 5.0.6 so plus I aint gonna use their system w/o docker so why do issue on their side when I tell when I run it via docker, they just tell to go open issue to the one who made the docker image in the first place. Been there, done that :)

Closing issue.

**Comment by blackandred** [comment]
FYI, another possibility could be the locale set up in the docker
container, by default it's probably en_US or just C. I know you closed the
issue - just a hint for the future.
